readme.md

```diff
@@ -0,0 +1,248 @@
+* Custom Post Type Support
+* Compatibility with WP 3.0 and 3.1
+* Gravatars
+* Lots and lots and lots of bug fixes
+* Thanks to everyone who submitted bugs, fixes, and suggestions! And for your patience!
+
+### 2.1.1 (Oct. 16, 2009) 
+* Fix for coauthors not being added if their username is different from display name
+* Fixes to readme.txt (fixes for textual and punctuation errors, language clarification, minor formatting changes) courtesy of [Waldo Jaquith](http://www.vqronline.org)
+
+### 2.1 (Oct. 11, 2009) 
+* Fixed issues related to localization. Thanks to Jan Zombik <zombik@students.uni-mainz.de> for the fixes.
+* Added set_time_limit to update function to get around timeout issues when upgrading plugin
+
+### 2.0 (Oct. 11, 2009) 
+* Plugin mostly rewritten to make use of taxonomy instead of post_meta
+* Can now see all authors of a post under the author column from Edit Posts page
+* All authors of a post are now notified on a new comment
+* Various javascript enhancements
+* New option to allow subscribers to be added as authors
+* All Authors can edit they posts of which they are coauthors
+* FIX: Issues with wp_coauthors_list function
+* FIX: Issues with coauthored posts not showing up on author archives
+
+### 1.2.0 (Jun. 16, 2012) 
+* FIX: Added compatibility for WordPress 2.8
+* FIX: Added new template tags (get_the_coauthor_meta & the_coauthor_meta) to fix issues related to displaying author info on author archive pages. See [Other Notes](http://wordpress.org/extend/plugins/co-authors-plus/other_notes/) for details.
+* FIX: Plugin should now work for plugins not using the 'wp_' DB prefix
+* FIX: Coauthors should no longer be alphabetically reordered when the post is updated
+* FIX: Plugin now used WordPress native AJAX calls to tighten security
+* DOCS: Added details about the new template tags
+
+### 1.1.5 (Apr. 26, 2009) 
+* FIX: Not searching Updated SQL query for autosuggest to search through first name, last name, and nickname
+* FIX: When editing an author, and clicking on a suggested author, the original author was not be removed
+* DOCS: Added code comments to javascript; more still to be added
+* DOCS: Updated readme information
+
+### 1.1.4 (Apr. 25, 2009) 
+* Disabled "New Author" output in suggest box, for now
+* Hopefully fixed SVN issue (if you're having trouble with the plugin, please delete the plugin and reinstall)
+
+### 1.1.3 (Apr. 23, 2009) 
+* Add blur event to disable input box
+* Limit only one edit at a time.
+* Checked basic cross-browser compatibility (Firefox 3 OS X, Safari 3 OS X, IE7 Vista).
+* Add suggest javascript plugin to Edit Page.
+
+### 1.1.2 (Apr. 19, 2009) 
+* Disabled form submit when enter pressed.
+
+### 1.1.1 (Apr. 15, 2009) 
+* Changed SQL query to return only contributor-level and above users.
+
+### 1.1.0 (Apr. 14, 2009) 
+* Initial beta release.
+
+## Installation 
+
+1. IMPORTANT: Please disable the original Co-Authors plugin (if you are using it) before installing Co-Authors Plus
```

Comment:
Odd, diff is showing this numbered as:
1.
1.
1.
1.
1.

Looks fine on viewing though!